routes/ping.js

```diff
@@ -2,7 +2,9 @@ const express = require('express')
 
 const router = new express.Router()
 const { pong } = require('../controllers/ping')
+const isAdmin = require('../middlewares/isAdmin')
 
 router.get('/', pong)
+router.get('/admin', isAdmin, pong)
```

Comment:
this route is only for testing purposes.. to see if the sever is online... please do not use middleware for admin access here...